LISISO/latestsupportedkernel

```diff
@@ -4,3 +4,4 @@ RHEL73:3.10.0-514.64.2.el6.x86_64
 RHEL74:3.10.0-693.47.2.el7.x86_64
```

Comment:
This file should be obsolete as we supported dynamic detection. For now please update the latest version for all distros.

Reply:
updated the latest versions for distros